endaq/calc/filters.py

```diff
@@ -4,10 +4,49 @@
 import functools
 
 import pandas as pd
+import numpy as np
 import scipy.signal
 
 from endaq.calc import utils
 
+def _get_filter_frequencies_type(low_cutoff, high_cutoff):
+    """Get the filter type and cutoff frequency array."""
+    cutoff_freqs: Union[float, Tuple[float, float]]
+    filter_type: str
+
+    filter_type = ""
+    if low_cutoff is not None and high_cutoff is not None:
+        cutoff_freqs = (low_cutoff, high_cutoff)
+        filter_type = "bandpass"
+    elif low_cutoff is not None:
+        cutoff_freqs = low_cutoff
+        filter_type = "highpass"
+    elif high_cutoff is not None:
+        cutoff_freqs = high_cutoff
+        filter_type = "lowpass"        
```

Comment:
Suggesting a change to make it behave like it did before

```suggestion
    if low_cutoff is not None and high_cutoff is not None:
        cutoff_freqs = (low_cutoff, high_cutoff)
        filter_type = "bandpass"
    elif low_cutoff is not None:
        cutoff_freqs = low_cutoff
        filter_type = "highpass"
    elif high_cutoff is not None:
        cutoff_freqs = high_cutoff
        filter_type = "lowpass"        
    else:
        return None, None
```